spec/presenters/account_show_presenter_spec.rb

```diff
@@ -157,6 +157,24 @@
 
       it { is_expected.to eq(true) }
     end
+
+    context 'when current user has ipp pending profile deactivated for password reset' do
+      let(:user) { create(:user, :with_pending_in_person_enrollment) }
+
+      it 'is expected to return false' do
+        account_show = AccountShowPresenter.new(
+          decrypted_pii: {},
+          sp_session_request_url: nil,
+          authn_context: nil,
+          sp_name: nil,
+          user: user.reload,
+          locked_for_session: false,
+        )
+
+        user.profiles.first.update!(deactivation_reason: :password_reset)
```

Comment:
Minor:

- With `context` describing the expected scenario for all tests in the block, we could move actual deactivation to either the `let(:user)` or a `before` block, rather than inline to each test. This'll make it easier if we want to add additional tests for this scenario down the line.
- Is `user.reload` strictly necessary?

Same points apply to the context below.

```suggestion
      let(:user) { create(:user, :with_pending_in_person_enrollment) }

      before do
        user.profiles.first.update!(deactivation_reason: :password_reset)
      end

      it 'is expected to return false' do
        account_show = AccountShowPresenter.new(
          decrypted_pii: {},
          sp_session_request_url: nil,
          authn_context: nil,
          sp_name: nil,
          user: user,
          locked_for_session: false,
        )

```

Reply:
I think I see what you mean. No, .reload was not actually needed so I took it out.